server/app/initializers/rollbar.rb

```diff
@@ -0,0 +1,9 @@
+if ENV['ROLLBAR_TOKEN'].to_s != ''
+  require 'rollbar'
+
+  Rollbar.configure do |config|
+    config.access_token = ENV['ROLLBAR_TOKEN']
```

Comment:
`config.environment = ENV['ROLLBAR_ENVIRONMENT'] || ENV['GRID_NAME']` would be my pick. It defaults to `unspecified`.